pom.xml

```diff
@@ -80,6 +80,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>apache-httpcomponents-client-4-api</artifactId>
+            <scope>test</scope>
```

Comment:
This was only needed to silence warnings in `InjectedTest` relating to `workflow-api-plugin/src/test/java/org/jenkinsci/plugins/workflow/DirectArtifactManagerFactory.java`.